assets/training/distillation/src/common/constants.py

```diff
@@ -69,16 +70,7 @@
 DEFAULT_TOP_P = 0.1
 DEFAULT_TEMPERATURE = 0.2
 
-# CHAIN OF THOUGHT (COT)
-COT_SYSTEM_PROMPT = (
-    "You are a helpful assistant. "
-    "Write out in a step by step manner your reasoning about the answer using no more than 80 words. "
-    "Based on the reasoning, produce the final answer. "
-    "Your response should be in JSON format without using any backticks. "
-    "The JSON is a dictionary whose keys are 'reason' and 'answer_choice'. "
-    "Always generate a syntactically correct JSON without using markdown and any additional words. "
-)
-
+# CHAIN OF THOUGHT (COD)
```

Comment:
nit: COD -> COT

Reply:
Removed comment. Author of COD_SYSTEM_PROMPT did not have add a comment and not sure what D stands for in COD